PolishWithBlondes/src/components/youtube-video/youtube.scss

```
@@ -0,0 +1,57 @@
+.youtube-videos {
+  display: flex;
+  flex-direction: row;
+  width: 1000px;
+  justify-content: space-between;
+}
+
+.video-card {
+  width: 480px;
+}
+
+.video-thumbnail-container {
+  position: relative;
+  width: 480px;
+  height: 250px;
+  overflow: hidden;
+  border-radius: 6px;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+}
+
+.video-thumbnail-container {
+  cursor: pointer;
+}
+
+.youtube-icon {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  z-index: 1;
+  left: 45%;
+  top: 40%;
```

That is not a good way to center an absolutely positioned compoenent. Should be:
```suggestion
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
```
This is the pattern for it. Now, no matter the size of the component you can be sure that it will be placed in the center